intranet/apps/eighth/management/commands/dev_create_blocks.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date, timedelta
+
+from django.core.management.base import BaseCommand, CommandError
+
+from intranet.apps.eighth.models import EighthActivity, EighthBlock, EighthScheduledActivity
+
+
+class Command(BaseCommand):
+    help = "Generate 8th Period Blocks"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("end_date")
+
+    def handle(self, *args, **kwargs):
+        try:
+            end_month, end_day, end_year = kwargs["end_date"].split("/")
+        except OSError as ex:
+            raise CommandError(str(ex)) from ex
+
+        wed_a_activities = EighthActivity.objects.filter(wed_a=True)
+        wed_b_activities = EighthActivity.objects.filter(wed_b=True)
+        fri_a_activities = EighthActivity.objects.filter(fri_a=True)
+        fri_b_activities = EighthActivity.objects.filter(fri_b=True)
+
+        curr_date = date.today()
+        end_date = date(int(end_year), int(end_month), int(end_day))
+        delta = timedelta(days=1)
+
+        while curr_date <= end_date:
+            if curr_date.weekday() == 2:
+                wed_ablock = EighthBlock.objects.get_or_create(date=curr_date, block_letter="A")[0]
+                wed_bblock = EighthBlock.objects.get_or_create(date=curr_date, block_letter="B")[0]
+                for activity in wed_a_activities:
+                    scheduled_activity = EighthScheduledActivity.objects.get_or_create(block=wed_ablock, activity=activity)[0]
+                    wed_ablock.activities.add(activity)
```

Comment:
Shouldn't this be:
```suggestion
                    wed_ablock.activities.add(scheduled_activity)
```

Reply:
The activities field in the EighthBlock model takes EighthActivity objects instead of EighthScheduledActivity objects

Reply:
Oops, my bad. The docstrings reflected a different reality.